internal/dependencies/tools/test_data/aws-cli/2.8.12/aws

```diff
@@ -0,0 +1,27 @@
+#!/opt/homebrew/Cellar/awscli/2.8.12/libexec/bin/python3.10
```

Comment:
this is macos only, it should be a bit more generic I think

Reply:
Also: what is this used for? perhaps a simpler shell script that outputs a hard-coded version as we did in other tests is good enough?

Reply:
this is the aws-cli's binary. We use the binary for the unit tests as other tools